HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ---------------
 
+2.9.0 (2020-06-17)
+++++++++
+- Fix exception handling for OAuth
+- Documentation updates 
```

Comment:
I think we can remove this and fix exception handling for OAuth. Not a bug fix or feature add users need to be aware of.

Reply:
I removed the documentation one and left the other one.  Since the exception being thrown has changed I thought there could possibly be an impact to users.

Reply:
The exception being thrown isn’t changing. Just whatever we name it in the SDK. I don’t believe the user will see the impact